cartridges/int_cloudinary/cartridge/templates/default/include/cloudinaryShrinkwrap.isml

```diff
@@ -4,15 +4,13 @@
 </isscript>
 
 <script>
-    document.addEventListener('DOMContentLoaded', function() {
-        var shrinkwrapUrl = JSON.parse('<isprint value="${JSON.stringify(pdict.CLDCoreShrinkwrapJSURL)}" encoding="jsonvalue"/>');
-        if (document.querySelector('script[src="'+shrinkwrapUrl+'"]') === null) {
-            const script = document.createElement('script');
-            script.src = shrinkwrapUrl;
-            document.body.appendChild(script);
-            script.onload = () => {
-                window.makeCloudinaryImagesResponsive();
-            };
-        }
-    });
+    var shrinkwrapUrl = JSON.parse('<isprint value="${JSON.stringify(pdict.CLDCoreShrinkwrapJSURL)}" encoding="jsonvalue"/>');
+    if (document.querySelector('script[src="'+shrinkwrapUrl+'"]') === null) {
+        const script = document.createElement('script');
+        script.src = shrinkwrapUrl;
+        document.body.appendChild(script);
+        script.onload = () => {
```

Comment:
why do we need this onload logic scripts? 
can't we just place window.makeCloudinaryImagesResponsive(); after a direct call to the shrinkWrap script?